spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:name) {|n| "User #{n}" }
+  end
+end
```

Comment:
Like that you know how to use these, but in a timed situation may be more trouble than they're worth. This seems like an investment that pays off in an application with a larger test suite.